src/Select-Forms-v5.php

```diff
@@ -0,0 +1,65 @@
+<?php namespace Lean\Acf\Types\Gravity;
+
+/**
+ * Setup configuration of the ACF fields form.
+ */
+class Acf_Field_Select_Form extends \acf_field {
```

Comment:
Basically:

- To follow the guideline from the ACF of the new definition of types where the -v`n` and type names are specified on the files names. 
- PSR4 is useless in this specific case, because the file is loaded with a hook and not by autoloading.